bundle-workflow/tests/tests_test_workflow/test_dependency_installer.py

```diff
@@ -0,0 +1,93 @@
+import os
+import shutil
+import unittest
+from unittest.mock import call, patch
+
+from manifests.build_manifest import BuildManifest
+from test_workflow.dependency_installer import DependencyInstaller
+
+
+class DependencyInstallerTests(unittest.TestCase):
+    def setUp(self):
+        self.maxDiff = None
+        self.manifest_filename = os.path.join(
+            os.path.dirname(__file__),
+            "../tests_assemble_workflow/data/opensearch-build-1.1.0.yml",
+        )
+        self.manifest = BuildManifest.from_path(self.manifest_filename)
+
+    @patch("test_workflow.dependency_installer.S3Bucket")
```

Comment:
you can replace this with S3Bucket that's checked into main

Reply:
This is the mocked version of the `S3Bucket` checked in main.